static_src/util/cf_api.js

```diff
@@ -247,8 +247,8 @@ export default {
   },
 
   fetchOrgs() {
-    return http.get(`${APIV}/organizations`)
-      .then(res => this.formatSplitResponses(res.data.resources))
+    return this.fetchAllPages('/organizations',
+      (res) => res)
```

Comment:
Honestly I would leave the parens out of this function. We have inconsistent usage of `(data) => data` and `data => data` throughout the app, and the consensus in the JS community is to omit them, especially for one-line functions. This can be seen in the airbnb js style guide, which is pretty much the gold standard for JS style-guides (and the one we use in this app)!